less/home-page.less

```diff
@@ -58,12 +85,33 @@
       }
 
       .btn {
+       .button_look();
+
+      //  :hover{
+      //   background-color:white;
+      //   color:@btn-bg-color;
+      //   } 
+        
+        @media @phone{
+          .button_look(100%,50px);
+          padding-top:15px;
+        }
       }
     }
```

Comment:
here, instead of having to select the .btn class again, you could nest your hover inside the first .btn using &:hover